sentry/src/main/java/io/sentry/TransactionFinishedCallback.java

```diff
@@ -0,0 +1,13 @@
+package io.sentry;
+
+import org.jetbrains.annotations.NotNull;
+
+public interface TransactionFinishedCallback {
+
+  /**
+   * Called when observed transaction finishes
+   *
+   * @param transaction the transaction
+   */
+  void onTransactionFinished(@NotNull ITransaction transaction);
```

Comment:
`l`: Maybe we can rename this to `execute` as the name of the interface already contains this information. Same applies for `SpanFinishedCallback`.
```suggestion
  void execute(@NotNull ITransaction transaction);
```

Reply:
since its an interface with a single method, the method name will never be used (cus u can do lambda or function interfaces), but it makes sense so I will rename it